ej1_2/Book.java

```diff
@@ -0,0 +1,75 @@
+package oopExercises.ej1_2;
+
+public class Book {
+	private String name;
+	private Author author;
+	private double price;
+	private int qtyInStock;
+
+	// Constructor
+
+	public Book(String name, Author author, double price) {
+		this.name = name;
```

Comment:
this(name, author, price, 0);
podría reemplazar todas estas lineas, y poner en default value 0 a stock